tutorials/roofit/rf515_hfJSON.py

```diff
@@ -0,0 +1,32 @@
+import ROOT
+
+# start by creating an empty workspace
+ws = ROOT.RooWorkspace("workspace")
+
+# the RooJSONFactoryWSTool is responsible for importing and exporting things to and from your workspace
+tool = ROOT.RooJSONFactoryWSTool(ws)
+
+# use it to import the information from your JSON file
+tool.importJSON("rf515_hfJSON.js")
+
+# now, you can easily use your workspace to run your fit (as you usually would)
+# the model config is named after your pdf, i.e. <the pdf name>_modelConfig
+model = ws.obj("main_modelConfig")
+result = model.GetPdf().fitTo(ws.data("observed"),
+                              ROOT.RooFit.Save(),
+                              ROOT.RooFit.GlobalObservables(model.GetGlobalObservables()))
+result.Print()
+
+# in the end, you can again write to json
+# the result will be not completely identical to the JSON file you used as an input, but it will work just the same
+tool.exportJSON("myWorkspace.js")
+
+# you can again import it if you want and check for closure
```

Comment:
```suggestion
# You can again import it if you want and check for closure
```

Please start all these inline comments with a capital letter, like in the other tutorials.